packages/taquito/src/contract/prepare.ts

```diff
@@ -339,3 +341,32 @@ export const createSmartRollupOriginateOperation = async ({
     parameters_ty: parametersType,
   } as RPCSmartRollupOriginateOperation;
 };
+
+export const createStakingOperation = async (
+  {
+    amount,
+    fee = DEFAULT_FEE.TRANSFER,
+    gasLimit = DEFAULT_GAS_LIMIT.TRANSFER,
+    storageLimit = DEFAULT_STORAGE_LIMIT.TRANSFER,
+    mutez = false,
+  }: StakingParams,
+  pkh: string,
+  entrypoint: StakingEntrypoint
```

Comment:
if `stakingParmas` has source and parameter that we don't need to pass them in outside of the first object

Reply:
Done for `source`.